zerver/lib/actions.py

```diff
@@ -5153,11 +5154,11 @@ def do_invite_users(user_profile: UserProfile,
 def do_get_user_invites(user_profile: UserProfile) -> List[Dict[str, Any]]:
     days_to_activate = settings.INVITATION_LINK_VALIDITY_DAYS
     active_value = getattr(confirmation_settings, 'STATUS_ACTIVE', 1)
+    revoked_value = getattr(confirmation_settings, 'STATUS_REVOKED', 2)
```

Comment:
I'm not what's the reason for ``active_value = getattr(confirmation_settings, 'STATUS_ACTIVE', 1)`` instead of using ``confirmation_settings.STATUS_ACTIVE``. but I think we should just do ``confirmation_settings.STATUS_REVOKED`` instead of the getattr thing - and then possible get rid of the getattr approach for STATUS_ACTIVE too, in another commit :thinking:

Reply:
Should I change the I'm not what's the reason for active_value = getattr(confirmation_settings, 'STATUS_ACTIVE', 1) to  confirmation_settings.STATUS_ACTIVE, everywhere that it's used?